db/seeds.rb

```diff
@@ -13,11 +14,130 @@
-  ['Erik Zakrzewski', 'Intern', 'Anything can be worked out.', '"Somewhere, something incredible is waiting to be known." -- Carl Sagan', '"Love all, trust a few, do wrong to none." -- William Shakespeare', 'TBA', 'TBA', 'TBA', 'I one day hope to help build video games and helping children less fortunate than most.', 'To present a complete application that meets Artemis standards and expectations.', 'Become a valuable part of a programming team and maybe even help with game creation.', 'Communication skills. Stay active with group at all working times.', 'Work on proper github practices.', 'Increase ability to explain self/problems properly to eliminate wasted time.', 'I plan on dedicating full time hours to this project with my team. Providing a full and functioning app at the end of this internship.', 'I will be striving to make my goal of bringing VR to children a reality.', 'Working with a local hospital.', 'Possibly working with a sponsor.', 'My mission is to bettter my self and my surroundings by persuing a better career. Part of doing this is taking satisfaction from my work and being happy with my job.', 'I want to be a part of a team who delivers an application that serves Artemis and its clients. I plan on doing my best to serve Artemis and my fellow interns.', 'I am able to jump into things I have never done before with confidence that I can figure it out.', 'I am very new to almost every aspect of this career path and I need to be reminded that I am doing it to the best of my current ability.', 'Feedback is welcome and most appreciated paired with possible solutions if applicable. I am not great at plenty of things and can only get better if taught the right way.', 'Taking Unity (game dev software) tutorials in my free time and attempting to start a project to bring VR experiences to children in a local hospital.', 'DateTime.new(2018,1,22)']
+  ['Jordan Clark',
```

Comment:
This looks terrifying, but it's just adding new lines. Nothing was really changed.